packages/flutter_driver/lib/src/driver/vmservice_driver.dart

```diff
@@ -169,10 +146,12 @@ class VMServiceFlutterDriver extends FlutterDriver {
     /// Looks at the list of loaded extensions for the current [isolateRef], as
     /// well as the stream of added extensions.
     Future<void> waitForServiceExtension() async {
-      final Future<void> extensionAlreadyAdded = isolateRef
-        .loadRunnable()
-        .then((VMIsolate isolate) async {
-          if (isolate.extensionRpcs.contains(_flutterExtensionMethodName)) {
+      await client.streamListen(vms.EventStreams.kIsolate);
```

Comment:
this will throw an exception if the stream has already been subscribed to

Reply:
This would be unexpected, since we unsubscribe at the end of this method.